iml-agent/src/action_plugins/lpurge.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::agent_error::ImlAgentError;
+
```

Comment:
nit

```suggestion
```

Reply:
I used to group different sources of imports: local (this crate), external (other crate), std. But apparently, in Rust there is no much sense in doing that.